packages/shorebird_cli/lib/src/shorebird_web_console.dart

```diff
@@ -0,0 +1,14 @@
+class ShorebirdWebConsole {
+  static Uri buildLink(String path) {
```

Comment:
```suggestion
  static Uri uri({required String path}) {
```

Don't feel strongly about the named parameter, but `build` isn't necessary IMO